### PR TITLE
[RW-5341][risk=low] Optionally include demographics in metrics export

### PR DESCRIPTION
**Pull request by calbach**
There is active work to move to a nightly reporting pipeline. For now though, this script is still being run manually to generate metrics. There is a high priority request for this information - however I do not wish to make it part of the standard export, since the nature of the data is sensitive.

### CODE REVIEW

**Comment by jaycarlton** [comment]
Thanks for jumping on this. I probably would've just done an ad hoc query and then merged it inside sheets with some horrendous formulas, or left the join as an exercise for the reader.

File: api/tools/src/main/java/org/pmiops/workbench/tools/ExportWorkspaceData.java [hunk 1]
```diff
@@ -269,6 +297,39 @@ private WorkspaceExportRow toWorkspaceExportRow(DbUser user) {
             .map(dateFormat::format)
             .orElse(""));
     row.setCreatorRegistrationState(user.getDataAccessLevelEnum().toString());
+    row.setDegrees(
+        Optional.ofNullable(user.getDegreesEnum()).orElse(ImmutableList.of()).stream()
+            .map(Degree::toString)
+            .collect(Collectors.joining(",")));
```
Comment (ericsong) [inline api/tools/src/main/java/org/pmiops/workbench/tools/ExportWorkspaceData.java:303]: nit: the rest of the multi value columns are separated with a newline as well

File: api/tools/src/main/java/org/pmiops/workbench/tools/ExportWorkspaceData.java [hunk 2]
```diff
@@ -269,6 +297,39 @@ private WorkspaceExportRow toWorkspaceExportRow(DbUser user) {
...
+    DbDemographicSurvey demo = user.getDemographicSurvey();
```
Comment (ericsong) [inline api/tools/src/main/java/org/pmiops/workbench/tools/ExportWorkspaceData.java:304]: nit: i think you can just move this object into the conditional and drop the null check

Comment (calbach) [reply api/tools/src/main/java/org/pmiops/workbench/tools/ExportWorkspaceData.java:304]: `includeDemographics` is driven by a flag - not null'ness of the demographics value. So this doesn't work - unless I'm misunderstanding the suggestion.

File: api/tools/src/main/java/org/pmiops/workbench/tools/ExportWorkspaceData.java (hunk 1 above)
Comment (calbach) [reply api/tools/src/main/java/org/pmiops/workbench/tools/ExportWorkspaceData.java]: Thanks, missed that.

**Pull request merged**